### PR TITLE
[NMS] Add support to configure AGW management network GW IP

### LINKED ISSUE #3254: NMS: Add support to configure AGW management network GW IP
we have new config AGW management interface, sgi_management_iface_gw. we need to add it to NMS UI. ref: https://github.com/magma/magma/blob/master/docs/readmes/howtos/ue_ip_address_management.md

### PR DESCRIPTION
**Pull request by HannaFar**
Signed-off-by: HannaFar <hannafarag159@gmail.com>

## Summary

Add support to configure AGW management network GW IP

Capture d’écran 2020-10-16 à 18 51 03

Capture d’écran 2020-10-16 à 18 51 49

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by andreilee — APPROVED** [review]

**Review by karthiksubraveti — COMMENTED** [review]
#3256 just merged which adds sgi_management_iface_gw attribute to the gateway epc struct. Can you rebase your branch and add a attribute for configuring that as well here. Thanks.

**Review by karthiksubraveti — CHANGES_REQUESTED** [review]
#3256 just merged which adds sgi_management_iface_gw attribute to the gateway epc struct. Can you rebase your branch and add a attribute for configuring that as well here. Thanks.

**Review by karthiksubraveti — COMMENTED** [review]
Also as @andreilee  mentioned in slack, Can you change the DNS field placeholders to show dummy IP addresses instead labels.

File: nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js [hunk 1]
```diff
@@ -707,6 +709,31 @@ export function EPCEdit(props: Props) {
...
+              placeholder="1.1.1.1/24"
+              type="string"
+              fullWidth={true}
+              value={EPCConfig.sgi_management_iface_static_ip}
+              onChange={({target}) =>
+                handleEPCChange('sgi_management_iface_static_ip', target.value)
+              }
+            />
+          </AltFormField>
+          <AltFormField label={'SGI VLAN ID'}>
```
Comment (karthiksubraveti) [inline nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js]: Can we display this only when the network is configured under bridged mode
https://github.com/magma/magma/blob/master/docs/readmes/howtos/ue_ip_address_management.md#agw-management-traffic.

Comment (HannaFar) [reply nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js]: it's disabled when NATed mode is enabled . Should I not display it instead ?

**Review by pshelar — CHANGES_REQUESTED** [review]
Lets rename label "Gateway SGi IP" to "SGi network Gateway IP address"
Also we need to make it required field when static IP is configured.

File: nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js [hunk 2]
```diff
@@ -707,6 +710,44 @@ export function EPCEdit(props: Props) {
...
+              placeholder="1.1.1.1"
+              type="string"
+              fullWidth={true}
+              value={EPCConfig.sgi_management_iface_gw}
+              onChange={({target}) =>
+                handleEPCChange('sgi_management_iface_gw', target.value)
+              }
+            />
+          </AltFormField>
+          <AltFormField label={'SGI Static IP'}>
```
Comment (pshelar) [inline nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js]: Lets rename "SGI Static IP" to "SGi management interface IP address"

File: nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js [hunk 3]
```diff
@@ -707,6 +710,44 @@ export function EPCEdit(props: Props) {
...
+              type="string"
+              fullWidth={true}
+              value={EPCConfig.sgi_management_iface_static_ip}
+              onChange={({target}) =>
+                handleEPCChange('sgi_management_iface_static_ip', target.value)
+              }
+            />
+          </AltFormField>
+          {!EPCConfig.nat_enabled && (
+            <AltFormField label={'SGI VLAN ID'}>
```
Comment (pshelar) [inline nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js]: Lets rename "SGI VLAN ID" to "SGi management network VLAN id"

**Review by pshelar — CHANGES_REQUESTED** [review]

File: nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js [hunk 4]
```diff
@@ -698,15 +707,56 @@ export function EPCEdit(props: Props) {
...
               onChange={({target}) =>
                 handleEPCChange('dns_secondary', target.value)
               }
             />
           </AltFormField>
+          <AltFormField label={'SGi network Gateway IP address'}>
+            <OutlinedInput
+              data-testid="gwSgiIP"
+              placeholder="1.1.1.1"
+              required={
```
Comment (karthiksubraveti) [inline nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js:723]: Nice. I didn't know OutlinedInput takes in a required parameter.

**Review by karthiksubraveti — APPROVED** [review]
Changes. LGTM. Thanks for fixing this

**Review by pshelar — APPROVED** [review]

**Pull request merged**